packages/terra-worklist-data-grid/tests/jest/Cell.test.jsx

```diff
@@ -1,26 +1,29 @@
 import React from 'react';
 /* eslint-disable-next-line import/no-extraneous-dependencies */
 import { shallowWithIntl } from 'terra-enzyme-intl';
+import { IntlProvider } from 'react-intl';
 import Cell from '../../src/subcomponents/Cell';
 
 describe('Cell', () => {
   it('verifies that only a row header cell (<th>) is created when isRowHeader prop is true', () => {
     const wrapper = shallowWithIntl(
-      <Cell
-        rowId="RowID"
-        columnId="ColumnId"
-        rowIndex={1}
-        columnIndex={2}
-        key="key"
-        isTabStop={false}
-        isSelected={false}
-        isRowHeader
-        onCellSelect={jest.fn}
-        height="50px"
-      >
-        Data in cell
-      </Cell>,
-    ).dive();
+      <IntlProvider locale="en">
```

Comment:
Is there a reason why the locale is being provided?

Reply:
Yes, because the component has been wrapped with React.memo.